p5/sketch/Vispy2DRenderer/base.py

```diff
@@ -86,6 +86,8 @@ def __init__(self, setup_method, draw_method,
         self._save_fname = 'screen'
         self._save_flag = False
 
+        builtins.frame_count = 0
```

Comment:
I'm not sure if this is necessary, since `frame_count` is also initialized here: https://github.com/p5py/p5/blob/69000548faf556a77e1cdad7f6633f2cdd47006a/p5/sketch/userspace.py#L58

Reply:
If there would be two `run` calls in the sketch , then the second run call will take the old `frame_count` values, so this has to be reset.

@niklasr22 instead of resetting it to `0` it should be reset to `-1`

Reply:
Ah I see, thanks for pointing this out!